src/sentry/runner/commands/killswitches.py

```diff
@@ -0,0 +1,91 @@
+import click
+import yaml
+
+from sentry.runner.decorators import configuration
+
+
+@click.group()
+def killswitches():
+    "Manage killswitches for ingestion pipeline."
+
+
+def _safe_modify(killswitch_name, modify_func):
+    from sentry import killswitches, options
+
+    option_value = options.get(killswitch_name)
+    new_option_value = modify_func(option_value)
+
+    if option_value == new_option_value:
+        click.echo("No changes!")
+        raise click.Abort()
+
+    click.echo("Before:")
+    click.echo(killswitches.print_conditions(option_value))
+    click.echo("After:")
+    click.echo(killswitches.print_conditions(new_option_value))
+
+    click.confirm("Should the changes be applied?", default=False, show_default=True, abort=True)
+    options.set(killswitch_name, new_option_value)
+
+
+@killswitches.command()
+@click.argument("killswitch", required=True)
+@configuration
+def edit(killswitch):
+    """
+    Edit killswitch conditions all at once using $EDITOR.
+
+    For a list of killswitches to edit, use `sentry killswitches list`.
+
+    For example:
+
+        sentry killswitches edit store.load-shed-pipeline-projects
+    """
+
+    from sentry import killswitches
+
+    def edit(option_value):
```

Comment:
I have mixed feelings about the "edit" interface. 
My main worry is how easy it is to fuck up YAML formatting, and then drop all events for the given project instead of transactions only. 
Thinking aloud here, maybe we should make some fields mandatory, and e.g. for event_type make a new option ALL?

Reply:
I was trying to make the user catch those mistakes with the confirmation prompt that shows alternative formatting in "SQL". I think we can make some fields mandatory but if we add more killswitches where we e.g. don't have event_type it would not make sense.

Basically we need stronger typing here, kinda tricky